README.rst

```diff
@@ -1,20 +1,17 @@
-BenchOpt bnechmark for Convolutional Sparse Coding
+BenchOpt benchmark for Convolutional Sparse Coding
 =====================
 |Build Status| |Python 3.6+|
 
 BenchOpt is a package to simplify and make more transparent and
 reproducible the comparisons of optimization algorithms.
 This benchmark is dedicated to solver of convolutional sparse coding:
 
-.. math::
+$$
```

Comment:
No need to convert the readme to .md to render maths equation (cf. [benchmark Lasso](https://github.com/benchopt/benchmark_lasso)).

I see that there is a problem with indentation
(also if I'm not mistaken: \ ----> \\)

Reply:
.rst seems to require escaping backslashes, from what I saw with markdown it was not the case hence it was better

also being able to break lines inside a math env for readability of source is interesting